### PR TITLE
CSS improvements

### PR DESCRIPTION
**Pull request by qti3e**
CSS:
- set a gray-white background for output canvas.
- add styles to edit title input element

### CODE REVIEW

**Comment by qti3e** [comment]
I do the same thing as you're saying, tale attention to `cors=false ` section in arguments list : )

File: src/im.ts [hunk 1]
```diff
@@ -96,9 +97,19 @@ function webImageDecoder(filename: string, mode: Mode)
...
+        console.info(`Trying to load image from CORS proxy...`);
+        img.src = "https://cors-anywhere.herokuapp.com/" + filename;
+      } else {
+        img.src = filename;
+      }
     } catch (e) {
-      resolve();
+      if (!cors) {
+        webImageDecoder(filename, mode, true).then(resolve);
+      }else {
```
Comment (ry) [inline src/im.ts]: style

File: website/main.scss [hunk 2]
```diff
@@ -144,6 +144,13 @@ header {
 
   .title-input {
     flex: 1 0 auto;
+    outline: none;
+    border: none;
+    height: 32px;
+    line-height: 32px;
+    border-bottom: solid 1px rgba(0, 0, 0, 0.5);
+
+    @include shadowHover;
```
Comment (ry) [inline website/main.scss]: Looks nice. Could you use the $size variables instead of pixels?

File: website/main.scss [hunk 3]
```diff
@@ -700,6 +707,30 @@ img.avatar {
...
+    linear-gradient(
+      45deg,
+      #efefef 25%,
+      transparent 25%,
+      transparent 75%,
+      #efefef 75%,
+      #efefef
+    );
+  background-position: 0 0, 10px 10px;
+  background-size: 21px 21px;
```
Comment (ry) [inline website/main.scss]: I wasn't able to notice these changes (generally I'd like to stay away from complex gradients like this until we're further along and can think about the design more cohesively)

File: src/im.ts [hunk 4]
```diff
@@ -75,7 +76,7 @@ export function toUint8Image(image: Tensor): Image {
   throw new Error(`Unsupported image rank.`);
 }
 
-function webImageDecoder(filename: string, mode: Mode)
+function webImageDecoder(filename: string, mode: Mode, cors = false)
```
Comment (ry) [inline src/im.ts]: It doesn't seem like there's a way to actually use the cors flag tho, since it's not surfaced at `imread()`.

Comment (qti3e) [reply src/im.ts]: it's a argument to block a potentially infinite loop

File: website/main.scss (hunk 3 above)
Comment (qti3e) [reply website/main.scss]: I used it instead of loading an image

File: website/main.scss (hunk 2 above)
Comment (qti3e) [reply website/main.scss]: ok

File: website/main.scss [hunk 5]
```diff
@@ -799,6 +806,30 @@ img.avatar {
   }
 }
 
+.output canvas {
```
Comment (ry) [inline website/main.scss]: Can you add a comment about what this does?

Comment (qti3e) [reply website/main.scss]: done

File: website/main.scss (hunk 3 above)
Comment (yungbeto) [reply website/main.scss]: styling lgtm 
can you share a quick screen grab of how it looks behind an image? @qti3e

Comment (qti3e) [reply website/main.scss]: image

Comment (yungbeto) [reply website/main.scss]: nice, so an image with some transparency would look something like this? -
image
@qti3e

Comment (qti3e) [reply website/main.scss]: @yungbeto yeah exactly

**Comment by ry** [comment]
LGTM included in #370

**Pull request closed without merging**